### PR TITLE
Instruction Updates

### PR DESCRIPTION
**Pull request by CooperCorona**
Added instructions to the README for initializing the project on a fresh machine, applying for GitHub student, and cloning the repo to a personal private repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,49 @@ Navigate to the project using the terminal, and then go to `app > tda-project-ui
 
 Install the necessary packages to run the project from that directory with the command `npm install`.
 
+Install the Angular command line interface with the command `npm install -g @angular/cli`. You may need to restart your terminal to use the `ng` command.
+
+Add npm global modules to your path. On Windows 10, search "path", which should bring up an option to "Edit the system environment variables". Select Environment Variables > Path > Edit. Select New and add C:\Users\<username>\AppData\Roaming\npm .
```
Comment (MatthewMeacham) [inline README.md]: This _theoretically_ happens with the NodeJS/NPM install--but I did include that SO link for this on line 12?

File: README.md [hunk 2]
```diff
@@ -15,6 +15,49 @@ Navigate to the project using the terminal, and then go to `app > tda-project-ui
...
+# Cloning the Repo
+Only 1 person needs to clone the repo per team. Once this section is done, the remaining members of the team will clone the new repository.
+
+Open Git Bash and navigate to the directory in which you want to place the project.
+
+Run `git clone https://github.com/raik383h/Spring2019CourseProjectTDA.git`. This clones the project on to your repository.
+
+Run `cd Spring2019CourseProjectTDA`.
+
+Run `git remote remote`. This disconnects your copy of the repository from the original repository.
```
Comment (MatthewMeacham) [inline README.md]: `git remove remote`? And is that command sufficient? Or do you need to specify origin or anything?

File: README.md [hunk 3]
```diff
@@ -15,6 +15,49 @@ Navigate to the project using the terminal, and then go to `app > tda-project-ui
...
+
+## Adding Team Members
+
+You have now successfully cloned the repo. Now you need to add your team members as collaborators so they can contribute code.
+
+Go to Settings > Collaborators in your GitHub repo (https://github.com/<username>/Spring2019CourseProjectTDA/settings/collaboration).
+
+Add each of your teammates' GitHub usernames and click "Add Collaborator". This sends an invitation to each team member. Team members will need to accept their invitations. If an invitation does not go through, you may need to cancel it and resend it.
+
+Once your team members have been added as collaborators, they can clone the repo to their machines by running `git clone https://github.com/<username>/Spring2019CourseProjectTDA.git`.
```
Comment (MatthewMeacham) [inline README.md]: Should we specify the `<username>` here is the username of whomever cloned the repo?

**Review by MatthewMeacham — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (CooperCorona) [reply README.md]: Ah yes you're right. I'll add a note that this step may not be necessary.

File: README.md (hunk 3 above)
Comment (CooperCorona) [reply README.md]: Yeah that's probably a good idea.

File: README.md (hunk 2 above)
Comment (CooperCorona) [reply README.md]: Whoops, you're right, that should be `git remote remove origin` (`remote` comes before `remove`).

**Pull request merged**